src/main/java/org/eclipse/golo/runtime/PropertyMethodFinder.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2012-2015 Institut National des Sciences Appliquées de Lyon (INSA-Lyon)
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.golo.runtime;
+
+import java.lang.invoke.MethodHandle;
+import java.lang.reflect.Field;
+
+import static java.lang.invoke.MethodHandles.*;
+import static java.lang.invoke.MethodType.methodType;
+
+public class PropertyMethodFinder extends RegularMethodFinder {
+
+  private static Object fluentSetter(Object o, Object notUsedSetterReturnValue) {
+    return o;
+  }
+
+  public PropertyMethodFinder(MethodInvocationSupport.InlineCache inlineCache, Class<?> receiverClass, Object[] args) {
+    super(inlineCache, receiverClass, args);
+  }
+
+  private MethodHandle fluentSetterMethodHandle() {
```

Comment:
Maybe this could be a static final field with static initialization, as the FALLBACK in `*Support`